README.md

```diff
@@ -13,10 +13,9 @@ This means you don't need to worry about which shell to use, nor file extensions
 
 ## Usage
 
-    $ ta-script <path_to_script> -- [...script_args]
+    $ ta-script <path_to_script> [...script_args]
```

Comment:
Used to require `--` to separate cli args from the args that belong to the script being called.  This is not intuitive and was forgotten multiple times in real use.  Used a more generic approach now that does not require `--`.